infrastructure/modules/platform/main.tf

```
@@ -97,3 +97,95 @@ resource "azurerm_application_insights" "app_insights" {
   resource_group_name = azurerm_resource_group.platform.name
   application_type    = "other"
 }
+
+resource "random_password" "password" {
```

I'm going to need another password as part of https://github.com/FutureNHS/futurenhs-platform/pull/61/files let's go with postgres_password and synapse_password or something?
```suggestion
resource "random_password" "synapse_password" {
```

---

This can generate multiple passwords.  You have ask it to dump an array rather than a string.  I originally had it, but @frigus02 asked me to remove because we currently don't have more than one password.  I'm happy to pair with you, when you have to create a new password.